annotations/views.py

```diff
@@ -139,13 +139,19 @@ def user_settings(request):
     """ User profile settings"""
 
     if request.method == 'POST':
-        form = UserChangeForm(request.POST)
+        form = UserChangeForm(request.POST,request.FILES)
         if form.is_valid():
-            for field in ['first_name', 'last_name', 'email']:
+            
+            
+            for field in ['first_name', 'last_name', 'email','imagefile']:
                 value = request.POST.get(field, None)
                 if value:
                     setattr(request.user, field, value)
             request.user.save()
+
+                #to save uploaded file
```

Comment:
Block comments should always be on the same level of indentation as the code that they are about. The hash should be followed by a space, and the comment should be grammatically correct: e.g. start with a caps letter, end with a period.